test/e2e_test.ts

```diff
@@ -37,17 +47,10 @@ export function checkClosureCompile(jsFiles: string[], externsFiles: string[], d
 }
 
 describe('golden file tests', () => {
-  it('generates correct Closure code', (done) => {
-    // Declaration tests do not produce .js files.
-    const tests = goldenTests().filter(t => !t.isDeclarationTest);
-    const goldenJs = ([] as string[]).concat(...tests.map(t => t.jsPaths));
-    goldenJs.push('src/closure_externs.js');
-    goldenJs.push('test_files/helpers.js');
-    goldenJs.push('test_files/clutz.no_externs/some_name_space.js');
-    goldenJs.push('test_files/clutz.no_externs/some_other.js');
-    goldenJs.push('test_files/import_from_goog/closure_Module.js');
-    goldenJs.push('test_files/import_from_goog/closure_OtherModule.js');
-    const externs = tests.map(t => t.externsPath).filter(fs.existsSync);
-    checkClosureCompile(goldenJs, externs, done);
+  it('compile with Closure (New Type Inference)', (done) => {
+    checkClosureCompile(true /* NTI */, done);
+  }, 15000 /* ms timeout */);
+  it('compile with Closure (Old Type Inference)', (done) => {
```

Comment:
We could do these in parallel maybe (?)

Reply:
I'm kind of scarred from gulp trying to run all kinds of stuff in parallel and spamming the console with the respective outputs. OTOH I think we capture all output here, so that won't be a problem. I'll give it a spin.